src/components/partial/repoPanel.vue

```diff
@@ -2,9 +2,6 @@
 <div>
```

Comment:
Better to change this component to:
```html
<template>
<div>
  <div class="field">
    <p class="control has-icons-left">
      <input v-model="filterInput" class="input" type="text" placeholder="输入名称查找镜像/反代服务">
      <span class="icon is-small is-left">
        <i class="fa fa-search"></i>
      </span>
    </p>
  </div>
  <repo-list :filter-input="filterInput" />
</div>
</template>
```

so that the input box occupies the whole row
![Screenshot from 2019-05-16 16-53-25](https://user-images.githubusercontent.com/8121231/57894325-56e73380-77fb-11e9-9c54-63cbaf0ce368.png)